syntax="proto2";
option cc_generic_services = true;

package peleus_demo.internal_service.syn;

message InterRequest {
    required string message = 1;
};

message InterResponse {
    required string message = 1;
};

service InterService {
    rpc query(InterRequest) returns (InterResponse);
};
